pkg/hive/resources.go

```diff
@@ -207,7 +208,7 @@ func adoptedClusterDeployment(namespace, clusterName, clusterID, infraID, locati
 			},
 			Platform: hivev1.Platform{
 				Azure: &hivev1azure.Platform{
-					BaseDomainResourceGroupName: "",
+					BaseDomainResourceGroupName: resourceGroupID,
```

Comment:
I guess if E2E passes then my question is answered, but are we sure that the ID goes here given that the parameter is attribute is call `...ResourceGroupName`?

I checked the doc comment in the `Platform` struct definition, but it didn't clarify.

Reply:
It's what we've been putting down on the "install" ClusterDeployment: https://github.com/Azure/ARO-RP/blob/d6d73be4c054f07e2eb440e3e3207de5dfa300c5/pkg/hive/install.go#L224. The fix here is to just update the "adopt" ClusterDeployment (which we apparently put down on Update/AdminUpdate) to do the same.

Reply:
I meant to put this context in the PR description but forgot.